gh-release/action.yaml

```diff
@@ -0,0 +1,67 @@
+name: GH Release
+description: Wrapper for softprops/action-gh-release
+
+inputs:
+  body:
+    description: "Note-worthy description of changes in release"
+    required: false
+  body_path:
+    description: "Path to load note-worthy description of changes in release from"
+    required: false
+  name:
+    description: "Gives the release a custom name. Defaults to tag name"
+    required: false
+  tag_name:
+    description: "Gives a tag name. Defaults to github.GITHUB_REF"
+    required: false
+  draft:
+    description: "Creates a draft release. Defaults to false"
+    required: false
+  prerelease:
+    description: "Identify the release as a prerelease. Defaults to false"
+    required: false
+  files:
+    description: "Newline-delimited list of path globs for asset files to upload"
+    required: false
+  fail_on_unmatched_files:
+    description: "Fails if any of the `files` globs match nothing. Defaults to false"
+    required: false
+  repository:
+    description: "Repository to make releases against, in <owner>/<repo> format"
+    required: false
+  token:
+    description: "Authorized secret GitHub Personal Access Token. Defaults to github.token"
+    required: false
+    default: ${{ github.token }}
+  target_commitish:
+    description: "Commitish value that determines where the Git tag is created from. Can be any branch or commit SHA."
+    required: false
+  discussion_category_name:
+    description: "If specified, a discussion of the specified category is created and linked to the release. The value must be a category that already exists in the repository. If there is already a discussion linked to the release, this parameter is ignored."
+    required: false
+  generate_release_notes:
+    description: "Whether to automatically generate the name and body for this release. If name is specified, the specified name will be used; otherwise, a name will be automatically generated. If body is specified, the body will be pre-pended to the automatically generated notes."
+    required: false
+  append_body:
+    description: "Append to existing body instead of overwriting it. Default is false."
+    required: false
+
+runs:
+  using: composite
+  steps:
+  - name: Release
+    uses: softprops/action-gh-release@v1
+    with:
```

Comment:
This looks to call directly to the gh-release action providing nothing extra.

This is going into `main` and `v0` keeps getting moved. 
How do we plan on ensuring backwards compatibility as the wrapped dependencies update?

Reply:
The point is instead of having softprops/action-gh-release@v1 in 20 or 30 repos, we have that here so if softprops/action-gh-release does a major release we only have one place we need to update.

The idea is that we make any changes required here at the abstration level, instead of updating every repo that uses this.  In the extreme case that we can't do some mapping to preserve compatibility, we would bump this repo to v1.0.0 and migrate caller repos as required.